.github/workflows/ci.yml

```diff
@@ -809,6 +812,8 @@ jobs:
         exclude:
           - python-version: ${{ github.event_name == 'pull_request_target' && '3.10' }}
           - python-version: ${{ github.event_name == 'pull_request_target' && '3.11' }}
+          - python-version: ${{ github.event_name == 'workflow_dispatch' && '3.10' }}
```

Comment:
This makes it so manually triggering these tests on a branch will also exclude 3.10 and 3.11 like on PRs.
This is useful for verifying changes because updates to ci.yml don't run on that PR, but only impact PRs after it merges.